### PR TITLE
Robot racer

### PR DESCRIPTION
**Pull request by epuzio**
finished robot racer, with conflicts

### CODE REVIEW

File: src/_00_IntroToArrays/RobotRacerNoRecipe.java [hunk 1]
```diff
@@ -7,27 +7,49 @@
...
-		Random randNum = new Random(100);
-		int randomNumber;
-		
-		String userAnswer = JOptionPane.showInputDialog("Place your bets! (Enter the number of robot you think will win the race)");
-		int numberOfWinner = Integer.parseInt(userAnswer) - 1;
-		
+		Random randNum = new Random();
+
+		String userResponse = JOptionPane.showInputDialog("How many robots do you want to race? (Between 1-9 robots, please.)");
+		try {
```
Comment (John-Ortega25) [inline src/_00_IntroToArrays/RobotRacerNoRecipe.java:21]: Why did you decide to put try and catch

File: src/_00_IntroToArrays/RobotRacerNoRecipe.java [hunk 2]
```diff
@@ -7,27 +7,49 @@
...
+			if (usersNumOfRobots > 5) {
+				robotList[i].moveTo((i * 90), 500);
+			}
+			robotList[i].setPenColor(randNum.nextInt(255), randNum.nextInt(255), randNum.nextInt(255));
 			robotList[i].penDown();
 			robotList[i].setSpeed(10);
 		}
 
-		while (robotWonRace() == false) {
+		while (!robotWonRace()) {
```
Comment (DCamaron1) [inline src/_00_IntroToArrays/RobotRacerNoRecipe.java]: Nicely done!

File: src/_00_IntroToArrays/RobotRacerNoRecipe.java [hunk 3]
```diff
@@ -7,27 +7,49 @@
...
 			for (int i = 0; i < robotList.length; i++) {
-				robotList[i].move(randNum.nextInt(30));
+				robotList[i].move(randNum.nextInt(70));
 			}
```
Comment (DCamaron1) [inline src/_00_IntroToArrays/RobotRacerNoRecipe.java]: Please remove this space

File: src/_00_IntroToArrays/RobotRacerNoRecipe.java [hunk 4]
```diff
@@ -7,27 +7,49 @@
...
+
+		String userResponse = JOptionPane.showInputDialog("How many robots do you want to race? (Between 1-9 robots, please.)");
+		try {
+			usersNumOfRobots = Integer.parseInt(userResponse);
+		} catch (NumberFormatException e) {
+			JOptionPane.showMessageDialog(null, "Letters are not acceptable values. Defaulting to 5 robots.");
+			usersNumOfRobots = 5;
+		}
+
+		while (usersNumOfRobots > 9 || usersNumOfRobots < 1) {
```
Comment (IsisJ) [inline src/_00_IntroToArrays/RobotRacerNoRecipe.java]: Magic numbers! create fancy description names for these guys 😄

File: src/_00_IntroToArrays/RobotRacerNoRecipe.java [hunk 5]
```diff
@@ -7,27 +7,49 @@
 import org.jointheleague.graphical.robot.Robot;
 
 public class RobotRacerNoRecipe {
-	static Robot[] robotList = new Robot[5];
+	private static int actualWinner;
```
Comment (John-Ortega25) [inline src/_00_IntroToArrays/RobotRacerNoRecipe.java:10]: Does it matter if its static or not

File: src/_00_IntroToArrays/RobotRacerNoRecipe.java (hunk 3 above)
Comment (epuzio) [reply src/_00_IntroToArrays/RobotRacerNoRecipe.java]: thanks!

File: src/_00_IntroToArrays/RobotRacerNoRecipe.java (hunk 4 above)
Comment (epuzio) [reply src/_00_IntroToArrays/RobotRacerNoRecipe.java]: 👍

**Pull request merged**